Tests/AlicerceTests/Logging/ConsoleLogDestinationTests.swift

```diff
@@ -38,7 +40,12 @@ class ConsoleLogDestinationsTests: XCTestCase {
 
         // preparation of the test subject
 
-        let destination = Log.ConsoleLogDestination(minLevel: .verbose, queue: queue)
+        let outputClosure: Log.ConsoleLogDestination.OutputClosure = {
+            [weak self] _ in self?.writtenLines += 1 }
```

Comment:
this could be done with an `expectation`, and configuring its `expectedFulfillmentCount` 😉